Task 2/2.2/Moveable.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _2._2
+{
+    public abstract class Moveable : Objects
+    {
+        public Moveable(char symbol, int x, int y) : base(symbol, x, y, false) { }
+
+        public int health = 2;
+        public int damage = 1;
+        public int stepLength = 1;
```

Comment:
Make it properties, don't forget about incapsulation. Classes shouldn't allow surprises, they should keep everything under control.
Plus you could fire death event from prop setter, when health goes below 0